conf/brs-default.properties

```diff
@@ -203,7 +203,12 @@ API.SSL_keyStorePassword = password
 
 # Database connection JDBC url
 # Append ;AUTO_SERVER=TRUE to enable automatic mixed mode access.
-DB.Url=jdbc:mariadb://localhost:3306/burstwallet
+# DB.Url=jdbc:mariadb://localhost:3306/burstwallet
```

Comment:
The intent is to make a release that simply works with zero changes. So standard is no mariadb, file based. On top of this there is the brs-template.properties which is installed as brs.properties for the user to edit. The new BurstGUI has a button allowing the edit the conf file directly.